packages/clay-date-picker/src/Helpers.ts

```diff
@@ -100,3 +102,19 @@ export function range({end, start}: {end: number; start: number}) {
 		(v, k) => k + start
 	);
 }
+
+/**
+ * Helper function for getting certain props from `moment`.
+ * This allows users to not have to import and use `moment` themselves.
+ */
+export function getLocaleProps(locale: string) {
```

Comment:
Just a thought, instead of using the `moment` instance directly, we can ask the user to pass that instance along with the `locale`, so we can have a better chance of removing the `moment` internally. What do you think?

Reply:
I think removing moment internally is probably better suited for a major release, so instead of going that route I thought it might be more helpful to just add some utilities that make it a bit easier for users so that they don't have to use moment themselves.

Reply:
Hmm yeah, makes sense.